mozilla/fx-private-relay
# Add Phone Dashboard [3/3] - Add Disabled + Empty Caller and SMS Senders Panel View

# New feature description

This adds 'disabled' and 'empty' states for the Caller and SMS senders panel view.

Default (Senders enabled)
![image](https://user-images.githubusercontent.com/13066134/185707675-54aaa281-1025-4e83-8194-7997af9c7754.png)

Default (Senders disabled)
<img width="1064" alt="image" src="https://user-images.githubusercontent.com/13066134/185707640-3cab3ff9-169a-471d-9a7d-8e0f390d6c6a.png">

Default (Empty logs / cleared history)
<img width="1009" alt="image" src="https://user-images.githubusercontent.com/13066134/185708253-c7132b6a-dcad-4627-9a92-f292aaa7366f.png">



Not applicable.

# How to test

If you have received relayed SMS/calls, the default logs view with the list of senders should be shown. If Caller & SMS logs are disabled in your settings, the disabled view panel should be shown. If there are no logs available (for example, if you've cleared your log history by disabling and then re-enabling the Caller & SMS settings option) then the empty state view should be shown.


# Checklist
